File: u1/Sample_Code/DemoTCPSender.java
import java.io.*;
import java.net.*;

public class DemoTCPSender{
	
	private static final String HOST_ADDR = "127.0.0.1";
	
	public static void main(String args[]){
		try{
			String _str; //declare variable for later use
			
			Socket _socket = new Socket(HOST_ADDR, 12345); //create the client socket
			PrintWriter _writer = new PrintWriter(_socket.getOutputStream(), true); //prepare the output stream
			
			BufferedReader _reader = new BufferedReader(new InputStreamReader(System.in));//prepare to get user input from standard input stream
			
			System.out.println("Type something or press Ctrl-C to end input");
			while((_str = _reader.readLine()) != null){//get user input
				_writer.println(_str);//write data to output stream
			}//repeat read and send user input
			
			_socket.close();//or close the connection
			System.out.println("Connection closed");
		}catch(UnknownHostException uhe){
			System.out.printf("Host [%s] not found\n", HOST_ADDR);
		}catch(IOException ioe){
			ioe.printStackTrace();
		}catch(Exception e){
			e.printStackTrace();
		}
	}
	
}

File: u1/Sample_Code/DemoTCPSender.py
import socket
import sys

def main():
	s = socket.socket(socket.AF_INET, socket.SOCK_STREAM) #create socket 
	s.connect(('127.0.0.1', 12345)) #make a connection request
	
	print('Type something or press Enter twice to end input')
	while True: #Repeating get user input and send
		line = sys.stdin.readline().strip() #Get user input
		if line:
			s.send(line.encode('utf-8')) #Sender send data to socket
		else:
			break
				
	s.close() # or close the connection
	print('Connection closed') #or close the connection

if __name__ == "__main__":
	main()